### DISCUSSION TITLE
Prebuilt configuration about nonce & timestamp attributes

### DISCUSSION THREAD

**Discussion by sylouuu** [spatie/laravel-webhook-server#73] [Ideas]
Hi,

What do you think/suggest about adding the support of nonce & timestamp attributes? This will prevent for replay attack and will add one more security layer.

We can imagine a way to enable/disable both of attributes and also a config value to specify the expiration time to validate the request.

It could be much appreciated in a next major release. Currently, I think we should create a custom signer to add conditions.

Best regards

**Comment by freekmurze** [comment]
I don't have time to invest in this name, but feel free to send a PR!
